### PR TITLE
feat(templates): added issue and pr templates

### PR DESCRIPTION
**Pull request by Aadarsh805**

### CODE REVIEW

**Comment by Aadarsh805** [comment]
@sboy99 have a look at the templates when you are free

**Comment by sboy99** [comment]
Can you provide some references of output, screenshot of a project using this template will be appreciated

**Comment by Aadarsh805** [comment]
> Can you provide some references of output, screenshot of a project using this template will be appreciated

yes sigmaTweet, https://github.com/Aadarsh805/SigmaTweet

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: ​🐞 Bug
+description: Report an issue to help us improve the project.
+title: "[BUG] <Title>"
+labels: ["Goal: fix"]
```
Comment (sboy99) [inline .github/ISSUE_TEMPLATE/bug.yml:4]: Add label "bug / report " as the default, we don't have "Goal: fix" label

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+name: ​🐞 Bug
+description: Report an issue to help us improve the project.
+title: "[BUG] <Title>"
```
Comment (sboy99) [inline .github/ISSUE_TEMPLATE/bug.yml:3]: Change [BUG] <Title> to  Bug: <Title>

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+name: Feature Request 💡
+description: Have any new idea or new feature for GitOpener? Please suggest! 
+title: "[Feature] <description>"
```
Comment (sboy99) [inline .github/ISSUE_TEMPLATE/feature_request.yml:3]: Change [Feature] <description>  to Feat: <Title>

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+labels: ["goal: addition"]
```
Comment (sboy99) [inline .github/ISSUE_TEMPLATE/feature_request.yml:4]: change "goal: addition" to "feat / request"

File: .github/ISSUE_TEMPLATE/other.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+name: Other 
+description: Use this for any other question or issue. Please do not create blank issues
+title: "[OTHER]"
+labels: ["status: awaiting triage"]
```
Comment (sboy99) [inline .github/ISSUE_TEMPLATE/other.yml:4]: remove `lebels`  for other request

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->
+
```
Comment (sboy99) [inline .github/pull_request_template.md:2]: add `closes #<issue_number>`

**Review by sboy99 — CHANGES_REQUESTED** [review]
Some of the labels need to be renamed, I have mentioned that. If possible  add a documentation issue template which adds `documentation` label by default

**Pull request merged**